### DISCUSSION TITLE
Documentation on sharing group AppStorage needs repair

### DISCUSSION THREAD

**Discussion by steve-h** [pointfreeco/swift-sharing#43] [Q&A]
I am setting up to use my group. user default so that I have access in a sharing extension and main app.

 I got a compile error using the code in the documentation as the init returns an Optional.
Not sure how to fail safe here. Using a force for now!

`@main struct EntryPoint: App {
  init() {
    prepareDependencies {
      $0.defaultAppStorage = UserDefaults(suiteName: "co.pointfree.suite")
    }
  }
  // ...
}`

**Comment by mbrandonw** [comment]
Hi @steve-h, would you be interested in PR'ing a change to the documentation to force unwrap the `UserDefaults`?

**Comment by steve-h** [comment]
Pull request done. I did the stack overflow research and it is safe. I also noticed in the example projects that the force unwrap was used. I also encountered some docc link warnings when building the documentation. I could not figure out how to fix them. (docc beginner)
